README.md

```diff
@@ -90,3 +90,33 @@ $ poetry run pytest tests_e2e
 ## Ansible
 
 This app can be managed across virtual machines using ansible. For more instructions, see the README in the ansible folder.
+
+## Docker
+
+This app is set up to run in docker, either in production mode or development mode.
+
+### Production
+
+You can build and run the app with docker in production mode.
+
+```bash
+$ docker build --target production --tag todo-app:prod .
+$ docker run -p 8080:5000 --env-file .env todo-app:dev
```

Comment:
Almost. I think you meant the second command to run "todo-app:prod" to match the tag used in the build command.